src/demo-app/input/input-container-demo.html

```diff
@@ -260,11 +260,11 @@
 <md-card class="demo-card demo-basic">
   <md-toolbar color="primary">Textarea Autosize</md-toolbar>
   <md-card-content>
-    <textarea md-autosize class="demo-textarea"></textarea>
+    <textarea md-textarea-autosize class="demo-textarea"></textarea>
```

Comment:
Should be `mdTextareaAutosize` (camelCase)

We should also make the old selector work at the same time as the new one for one release. 
(so both `md-autosize` and `mdTextAreaAutosize`)